tests/algorithms/test_EC.py

```diff
@@ -45,11 +45,68 @@ def test_invalid_algorithm(self):
         with pytest.raises(JWKError):
             ECDSAECKey({'kty': 'bla'}, ALGORITHMS.ES256)
 
+    def test_EC_jwk(self):
```

Comment:
I know that it makes for some duplicate code, but can you split this test into separate tests?  I get much more value out of seeing every test that fails in a single test run instead of fixing one portion of a test only to see another portion has failed. 

I see this as:
* `test_ECDSAECKey_public_jwk`
* `test_ECDSAECKey_private_jwk`
* `test_ECDSAECKey_invalid_jwk`
* `test_CryptographyECKey_public_jwk`
* `test_CryptographyECKey_private_jwk`
* `test_CryptographyECKey_invalid_jwk`